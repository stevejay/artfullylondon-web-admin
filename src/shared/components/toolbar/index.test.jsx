import React from 'react'

import Toolbar from '_src/shared/components/toolbar'

it('should render correctly', () => {
  const wrapper = shallow(<Toolbar><div id='child' /></Toolbar>)
  expect(wrapper).toMatchSnapshot()
})
