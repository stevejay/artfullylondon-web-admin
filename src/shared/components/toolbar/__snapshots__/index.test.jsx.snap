// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="src-shared-components-toolbar-index__toolbar"
  role="menubar"
>
  <div
    id="child"
  />
</div>
`;
